modules/graftorio/require.lua

```diff
@@ -0,0 +1,23 @@
+local Commands = require("expcore.commands")
+local config = require("config.graftorio")
+
+local general = require("modules.graftorio.general")
+local statics = require("modules.graftorio.statics")
+local forcestats = {}
+if config.modules.forcestats then
+	forcestats = require("modules.graftorio.forcestats")
+end
+
+Commands.new_command("collectdata", "Collect data for RCON usage")
+		:add_param("location", true)
+		:register(function()
+			-- this must be first as it overwrites the stats
+			-- also makes the .other table for all forces
+			statics.collect_statics()
+			general.collect_other()
+			if config.modules.forcestats then
+				forcestats.collect_production()
+			end
```

Comment:
`collect_loginet` is never called which is part of the `forcestats` module.
`general` has a config which is never tested for.
```suggestion
			statics.collect_statics()
			if config.modules.general then
			  general.collect_other()
			end
			if config.modules.forcestats then
				forcestats.collect_production()
				forcestats.collect_loginet()
			end
```